SonarQube.Common/Interfaces/LoggerVerbosity.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarQube.Common
+{
+    /// <summary>
+    /// Level of detail for the log messages. 
+    /// </summary>
+    /// <remarks>
+    /// Does not cover warnings and errors.
+    /// The levels are in step with the SonarQube verbosity levels (http://docs.sonarqube.org/display/SONAR/Server+Log+Management):
+    /// Info, Debug (for advanced logs), Trace (for advanced logs and logs that might have a perf impact)
+    /// </remarks>
+    [Flags]
```

Comment:
Does this need to be a Flags enum? Aren't we always going to log "Info" messages; isn't the only question whether we log "Debug" messages or not?